### PR TITLE
Add multiple workers configuration

### PR DESCRIPTION
**Pull request by ignas-gustainis**
@titas @laurynas 
Almost done, just have to figure out how to handle the config file properly (currently, it's filename is hard coded as `config/sidekiq-pool.yml`)

This PR will allow users to add several different workers - each could have different command line options - in a single sidekiq pool.

Any suggestions/comments?

Example config file:
```yaml
:workers:
  -
    :command: '-q crm -q default'
    :amount: 1
  -
    :command: '-q high'
    :amount: 1
```

### CODE REVIEW

File: lib/sidekiq/pool/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Sidekiq
   module Pool
-    VERSION = '0.1.7'
+    VERSION = '0.2.0'
```
Comment (laurynas) [inline lib/sidekiq/pool/version.rb]: I suggest bumping major version (1.0.0)

File: lib/sidekiq/pool/cli.rb [hunk 2]
```diff
@@ -5,6 +5,8 @@ module Sidekiq
   module Pool
     class CLI < Sidekiq::CLI
       def initialize
+        @settings = YAML.load_file('config/sidekiq-pool.yml')
```
Comment (laurynas) [inline lib/sidekiq/pool/cli.rb]: Maybe pool config file path could be passed by optional param with default to `config/sidekiq-pool.yml`?

**Comment by titas** [comment]
Oh, also modify readme accordingly

File: lib/sidekiq/pool/cli.rb [hunk 3]
```diff
@@ -151,42 +145,25 @@ def handle_master_signal(sig)
...
           logger.info "Sending #{sig} signal to the pool"
           signal_to_pool(sig)
         end
       end
 
-      def add_child
-        logger.info 'Adding child'
-        fork_child
+      def add_child(*arg)
+        puts "Adding child with args: #{arg}"
```
Comment (laurynas) [inline lib/sidekiq/pool/cli.rb]: Use `logger.info` instead of `puts`

Comment (laurynas) [inline lib/sidekiq/pool/cli.rb]: Use `logger.info` instead of `puts`

File: lib/sidekiq/pool/cli.rb [hunk 4]
```diff
@@ -129,13 +121,15 @@ def trap_signals
...
-        @pool << fork do
+      def fork_child(command)
+        pid = fork do
+          setup_options(command.split)
           @self_write.close
           $0 = 'sidekiq starting'
           options[:index] = @child_index++
           run_child
         end
+        @pool << {pid: pid, command: command}
```
Comment (laurynas) [inline lib/sidekiq/pool/cli.rb]: I suggest to add spaces when defining hash:
```ruby
@pool << { pid: pid, command: command }
```

**Comment by laurynas** [comment]
Nice!

**Comment by ignas-gustainis** [comment]
Changed readme, fixed the code according to Laurynas's comments, is there anything left to address?

**Pull request merged**